### PR TITLE
30/social impact people

### LINKED ISSUE #30: 🎨  Create Social Impact people section
#  Create Social Impact people section
---

#### Note: This issue is dependent on #31, please complete that issue first.

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want a summary in numbers of the organisation impact

### Description

This section contains an auto incrementing figure of the organisation impact, the increment starts from 0 to the provided figure in about 5secs. 

Then section has the following:

```
1. 4 side by side impact figures
2. A description section with an image floated left (reuse component on #31 here).

```

The figure counters need to increment in uniform and with ease

Screenshot 2023-02-06 at 10 15 23

Your assignment is to implement the component to appear as above.

Please take note of all the design features (including mobile)

#### Link to full design:
```
https://xd.adobe.com/view/7b72572b-b40a-4618-bfa6-5500ab28f13b-2e21/screen/31438bb0-400a-4da8-a374-c0440535c37c

mobile 
-

```
### Technical Analysis

This component will be created in libs/pages/elewa/social-impact

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 50 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the main style.scss
4. The code is isolated to the libs/page/home package, unless a justified reason is specified.
5. The component is aligned using CSS flex
6. The design is responsive for screen, tablet and phone
7. The image renders fast and has multiple srcsets

### PR DESCRIPTION
**Pull request by munenebrian**
# Description
Worked on the summary made sure the numbers run and stop at the specified ones.

Fixes #30 
## Type of change
- [ ] New feature (non-breaking change which adds functionality)
Screenshot

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
The description section is missing, also please link a video showing the increment as it happens. but good job so far 👍

File: libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts [hunk 1]
```diff
@@ -12,21 +12,20 @@ import { NextDirective } from './directives/next.directive';
 import { PrevDirective } from './directives/prev.directive';
 
 import { AboutUsRoutingModule } from './about-us.routing';
+import { HistoryCarouselComponent } from './components/history-carousel/history-carousel.component';
+import { ElewaGroupImageAndTextBannerComponent } from 'libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts:16]: Import from the aliased path

File: libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts [hunk 2]
```diff
@@ -12,21 +12,20 @@ import { NextDirective } from './directives/next.directive';
...
     TeamMembersCarouselComponent,
     NextDirective,
     PrevDirective,
     AboutUsCultureComponent,
     AboutUsPageComponent,
+    HistoryCarouselComponent,
+    ElewaGroupImageAndTextBannerComponent
   ],
-  exports: [TeamMembersCarouselComponent],
+  exports: [TeamMembersCarouselComponent, HistoryCarouselComponent],
```
Comment (LemmyMwaura) [inline libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts:29]: No need to export this component

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Provide mobile screenshots as well.

The figures at the top should be a row in pairs of 2 on mobile

File: libs/pages/elewa/about-us/src/lib/pages/about-us-page/about-us-page.component.html [hunk 3]
```diff
@@ -5,6 +5,8 @@
 
       <elewa-group-about-us-culture></elewa-group-about-us-culture>
       <elewa-group-team-members-carousel></elewa-group-team-members-carousel>
+      <elewa-group-history-carousel></elewa-group-history-carousel>
+      <elewa-group-elewa-group-image-and-text-banner></elewa-group-elewa-group-image-and-text-banner>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/about-us/src/lib/pages/about-us-page/about-us-page.component.html:9]: This should be in the same component with the figures, not here.

File: libs/pages/elewa/about-us/src/lib/components/history-carousel/history-carousel.component.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    </div>
+    <div class="counter-col">
+        <h2>{{projectcount}}+</h2>
+        <p class="count-text ">Projects finished</p>
+    </div>
+    <div class="counter-col">
+        <h2>{{investorcount}} </h2>
+        <p class="count-text ">Investors</p>
+    </div>
+</div>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/about-us/src/lib/components/history-carousel/history-carousel.component.html:18]: This is not a carousel, rename component to social-impact-enterprise-section

should contain figures and reusable (text and image) banner as well. With correct text as shown in design

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
sync

**Comment by IanOdhiambo9** [comment]
I'm closing PR because this issue has been closed.

**Pull request closed without merging**